examples/all-clusters-app/all-clusters-common/all-clusters-app.matter

```diff
@@ -235,24 +235,16 @@ server cluster BinaryInputBasic = 15 {
 }
 
 server cluster Binding = 30 {
-  readonly global attribute int16u clusterRevision = 65533;
-
-  request struct BindRequest {
-    NODE_ID nodeId = 0;
-    GROUP_ID groupId = 1;
-    ENDPOINT_NO endpointId = 2;
-    CLUSTER_ID clusterId = 3;
+  struct TargetStruct {
+    FABRIC_IDX fabricIdx = 0;
+    nullable NODE_ID nodeId = 1;
+    nullable GROUP_ID groupId = 2;
+    nullable ENDPOINT_NO endpointId = 3;
+    nullable CLUSTER_ID clusterId = 4;
```

Comment:
These are supposed to be optional, not nullable.

How did tests not catch this?  For that matter, _are_ there any tests being added here?  There should be.

Happy to review the details once that's fixed.

Note that for _storage_ purposes we don't have to store the Optionals (which use more space than just the value).  We can use the storage we have right now, in fact (to keep scope down in this PR), and a followup to reduce it by using a union appropriately if we want.

Reply:
I mistook the usage of nullable for optional in the zap templates and tests were conducted with null values. Has updated defnition and the test instructions.